apps/src/gamelab/blocks.js

```diff
@@ -211,5 +211,34 @@ export default {
         { name: 'COLOR', type: blockly.BlockValueType.COLOUR },
       ],
     });
+
+    createJsWrapperBlock({
+      color: EVENT_COLOR,
+      func: 'clickedOn',
+      blockText: 'when {SPRITE} clicked on',
+      args: [
+        { name: 'SPRITE', type: SPRITE_TYPE },
+      ],
+      eventBlock: true,
+    });
+
+    createJsWrapperBlock({
+      color: SPRITE_COLOR,
+      func: 'setPosition',
+      blockText: 'move {THIS} to the {POSITION} position',
+      args: [
+        {
+          name: 'POSITION',
+          options: [
+            ['top left', '"top left"'],
+            ['top center', '"top center"'],
+            ['bottom right', '"bottom right"'],
+            ['center right', '"center right"'],
+            ['random', '"random"'],
```

Comment:
Nooooo 😭

Reply:
?

Reply:
Haha, Josh is not a fan of this dropdown. We'd like to build a better alternative for setting sprite locations (e.g. dragging labeled pins onto the playspace), but I'd just like to get this out for Kiki to prototype with.

Reply:
Yep 😄.  No objections to adding this for prototyping, but I really hope we don't ship with this.

We want students to be thinking with more precision than 9 pre-set locations.  If I can set a sprite to `left-center`, how do I set it to halfway between `left-center` and `left-top`?